config/config.exs

```diff
@@ -30,7 +30,7 @@ config :media,
   otp_app: :your_app,
   active_database: "mongoDB",
   repo: :mongo,
-  router: YouAppWeb.Router,
+  # router: YouAppWeb.Router,
```

Comment:
if this is not needed, why do you keep it in the code ?

Reply:
removed @jfayad